test/unittest/rtps/builtin/BuiltinDataSerializationTests.cpp

```diff
@@ -1197,6 +1198,67 @@ TEST(BuiltinDataSerializationTests, contentfilterproperty_interoperability)
     ASSERT_EQ("200", out.content_filter().expression_parameters[3].to_string());
 }
 
+/*!
+ * \test Test for deserialization of messages with the maximum size of expression parameters
+ */
+TEST(BuiltinDataSerializationTests, contentfilterproperty_max_parameter_check)
```

Comment:
Maybe a negative test case can be included testing what happens when trying to serialize more than 100 parameters.